aries_cloudagent/config/logging.py

```diff
@@ -1,25 +1,25 @@
 """Utilities related to logging."""
 import asyncio
+from datetime import datetime, timedelta
```

Comment:
These imports ought to be groups to avoid re-ordering...

Reply:
I've considered adding `isort` or similar to pre-commit hooks and our format checks lol so it's handled automatically like black for formatting.